src/components/AutocompleteTypeField.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import Autocomplete from 'react-autocomplete';
-import '../css/Autocomplete.css';
+import '../CSS/Autocomplete.css';
```

Comment:
@tmccoy529 what is this showing as you, because i see the lowercase css changes in my branches.

Reply:
yeah mine is lower case also. Not sure why this got reverted back. Maybe a bad merge?